util/file_util.cc

```diff
@@ -62,6 +63,9 @@ Status CopyFile(Env* env, const std::string& source,
     }
     size -= slice.size();
   }
+  if (sync_after_copy) {
+    dest_writer->Sync(true);
+  }
```

Comment:
This needs to take an option of options.use_fsync. I know it is confusing but if the parameter is true, we'll call "fsync()", otherwise we'll call "fdatasync()". User provides what to call. The problem is, DB option is not available here, so we don't know which one we should call. The caller may  need to pass that information here too if we want to enable fsync.